api/v1beta1/infraenv_types.go

```diff
@@ -89,13 +89,13 @@ type InfraEnvSpec struct {
 	// +optional
 	CpuArchitecture string `json:"cpuArchitecture,omitempty"`
 
-	// IPXEScriptType the script type that should be served (HostRedirect/Boot)
-	// When the value is HostRedirect, the served script consists from URL redirection (i.e chain <url>).  The
+	// IPXEScriptType the script type that should be served (BootOrderControl/DiscoveryImageAlways)
+	// When the value is BootOrderControl, the served script consists from URL redirection (i.e chain <url>).  The
```

Comment:
I like the description from the git commit message better. "Boot from discovery ISO depending on the state of
the host." The stuff about the mac address and redirection could be confusing I think and is just an implementation detail. I suggest listing `DiscoveryImageAlways` first, since it's the simple and default case. Then something like this:

> When the value is BootOrderControl, the served ipxe script will look for an Agent record that matches the host's MAC address; if found, and if that Agent is in a state where it is provisioned and attached to a cluster, then the IPXE script will boot the host from disk. Otherwise it will boot the discovery ISO using the same script as the `DiscoveryImageAlways` option.

Reply:
@mhrivnak So do you want to take the text from the commit message, or what you wrote above ?